config/lava/lc-compliance/lc-compliance.jinja2

```diff
@@ -0,0 +1,19 @@
+- test:
+    timeout:
+      minutes: 15
```

Comment:
This job timed out:
https://lava.collabora.co.uk/scheduler/job/4005966

This job was resubmitted with a 60min timeout, it completed in 11min:
https://lava.collabora.co.uk/scheduler/job/4006094

So I think 30min would be good, to allow some headroom for when jobs run a bit slower.  NFS jobs duration partly depends on network and dispatcher I/O.

Reply:
Please also set the total job timeout to 45min or more.